Explorer/Assets/DCL/Web3Authentication/DappWeb3Authenticator.cs

```diff
@@ -0,0 +1,214 @@
+using Cysharp.Threading.Tasks;
+using DCL.Browser;
+using SocketIOClient;
+using SocketIOClient.Newtonsoft.Json;
+using SocketIOClient.Transport;
+using System;
+using System.Threading;
+
+namespace DCL.Web3Authentication
+{
+    public class DappWeb3Authenticator : IWeb3Authenticator
+    {
+        private const string SERVER_URL_DEV = "https://auth-api.decentraland.zone";
```

Comment:
I think we can move these settings to `DynamicSceneLoaderSettings` to a separate structure so we will have better control for testing servers.

Reply:
I think we could have a completely different scene to start the real application flow, decoupling the authentication from the dynamic scene loader. I think it could be useful for daily development to keep dynamic scene loader as simple as possible